test/modules/Math/signbit-test.chpl

```diff
@@ -0,0 +1,16 @@
+use Math;
+
+var x = -0.0;
+var y = 0.0;
+
+writeln(signbit(x:real(32)));
+writeln(signbit(y:real(32)));
+
+writeln(signbit(-NAN));
+writeln(signbit(NAN));
+
+writeln(signbit(x:real(64)));
+writeln(signbit(y:real(64)));
+
+writeln(signbit(-0));
+writeln(signbit(+0));
```

Comment:
I think this needs to be `-0.0` and `+0.0`. As is, it is effectively `signbit(-(0:int(64))` (the negation of the integer 0, instead of being negative 0.0 as a real)